spec/controllers/flood_risk_engine/enrollments/steps_controller/check_your_answers_spec.rb

```diff
@@ -45,11 +45,11 @@ def row_t(row_name, key)
       end
 
       it "table summary" do
-        expect(response.body).to have_tag :caption, text: /#{t(:table_summary)}/
+        expect(response.body).to have_tag :span, text: /#{t(:table_summary)}/
       end
     end
 
-    describe "rows" do
+    xdescribe "rows" do
```

Comment:
I marked them pending because we haven't tackled the changes to the Check your answers page. The ticket is in To Do: https://eaflood.atlassian.net/browse/RUBY-1464